Lib/multiprocessing/managers.py

```diff
@@ -167,8 +172,10 @@ def serve_forever(self):
         process.current_process()._manager_server = self
         try:
             accepter = threading.Thread(target=self.accepter)
-            accepter.daemon = True
+            watcher = threading.Thread(target=self._track_parent)
```

Comment:
Can simply launch the watcher only if `parent_process() is not None`.

Reply:
I added a new guard, but left the one in `_track_parent` (does not hurt)